app/code/Magento/Review/Api/GetProductRatingSummaryInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Review\Api;
+
+/**
+ * Interface to get product rating summary
+ */
+interface GetProductRatingSummaryInterface
```

Comment:
Please mark it as @api

Reply:
what existing API this new one substituting with?  The old one should be deprecated in its favor